frontend/packages/knative-plugin/src/components/overview/EventPubSubSubscribers.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import { Grid, GridItem } from '@patternfly/react-core';
+import { referenceFor } from '@console/internal/module/k8s';
+import { SidebarSectionHeading, ResourceLink } from '@console/internal/components/utils';
+import { Subscriber } from '../../topology/knative-topology-utils';
+import FilterTable from './FilterTable';
+import EventPubSubExpandButton from './EventPubSubExpandButton';
+import './EventPubSubSubscribers.scss';
+
+type EventPubSubSubscribersProps = {
+  subscribers?: Subscriber[];
+};
+
+const EventPubSubSubscribers: React.FC<EventPubSubSubscribersProps> = ({ subscribers }) => {
+  const [expanded, setExpanded] = React.useState('');
+
+  return (
+    <>
+      <SidebarSectionHeading text="Subscribers" />
+      {subscribers.length > 0 ? (
+        <ul className="list-group kn-event-subscriber-list">
+          {subscribers.map((sub) => (
+            <li className="list-group-item list-group kn-event-subscriber-list-item" key={sub.name}>
+              <ResourceLink kind={referenceFor(sub)} name={sub?.name} namespace={sub?.namespace} />
```

Comment:
nit: do we need check for `sub?` here and below as it'll be always defined
```suggestion
              <ResourceLink kind={referenceFor(sub)} name={sub.name} namespace={sub.namespace} />
```